### PR TITLE
Update and finish (some still pending) Italian translations

### PR DESCRIPTION
**Pull request by DepaMarco**
@7omate Hi. Please, have a look to my comments, inside the .po file

### CODE REVIEW

File: src/pydio/res/i18n/it.po [hunk 1]
```diff
@@ -176,9 +176,10 @@ msgstr "Riscontrato errore Proxy! Controllare i log..."
 msgid "Connection error: too many redirects"
 msgstr "Errore di connessione: troppe redirezioni"
 
+# I used "pacchetti" (packages, in English) because of "chunk" is not pretty translated into Italian and seems not related with Internet stuffs.
 #: job/continous_merger.py:685 ui/web_api.py:375
 msgid "Chunked encoding error, please check the logs"
-msgstr ""
+msgstr "Errore nella codifica dei pacchetti. Controlla i log."
```
Comment (7omate) [inline src/pydio/res/i18n/it.po]: Thanks for the explanation.

File: src/pydio/res/i18n/it.po [hunk 2]
```diff
@@ -783,19 +787,19 @@ msgstr "Nome File"
...
 
 #: res/i18n/html_strings.py:150
 msgid "Set restrictions on log file size here."
-msgstr ""
+msgstr "Limita la dimensione del file di log quì."
 
 #: res/i18n/html_strings.py:151
 msgid "Enhance the log details as you need them."
-msgstr ""
+msgstr "Arricchisci il dettaglio dei log come ritieni più opportuno"
```
Comment (7omate) [inline src/pydio/res/i18n/it.po]: Missing "."

File: src/pydio/res/i18n/it.po [hunk 3]
```diff
@@ -946,81 +951,86 @@ msgstr "Aggiungere una sincronizzazione"
...
+msgstr "Gestione Link"
 
 #: res/i18n/html_strings.py:213
+#, fuzzy
 msgid ""
 "If you want a durable and pretty link (like https://.../my-share-link), you "
 "can use this field. Link handle will be generated if left empty."
 msgstr ""
+"Se vuoi un link riconoscibile e duraturo (come https://.../mio-link-"
+"condivisione), puoi utilizzare questo campo. Un --<non so cosa>-- verrà "
```
Comment (7omate) [inline src/pydio/res/i18n/it.po]: Un link casuale

File: src/pydio/res/i18n/it.po [hunk 4]
```diff
@@ -1100,17 +1112,23 @@ msgstr "Autenticazione HTTP-Base fallita. Credenziali errate?"
 msgid "Token-based authentication failed, reload credentials?"
 msgstr "Autenticazione basata su Token fallita. Ricaricare le credenziali?"
 
+# Are you sure that is a question? Anyway I translated it into Italian...
```
Comment (7omate) [inline src/pydio/res/i18n/it.po]: It's not really a question but should only be seen in the logs...

File: src/pydio/res/i18n/it.po [hunk 5]
```diff
@@ -1100,17 +1112,23 @@ msgstr "Autenticazione HTTP-Base fallita. Credenziali errate?"
...
 #: sdkremote/exceptions.py:60
 msgid ""
 "Token-based authentication seems unsupported, this may impact performances?"
 msgstr ""
+"L'autenticazione basata su Token sembra non essere supportata, questo può "
+"intaccare le performance?"
 
+# I get an error on parameter %(filesize)
```
Comment (7omate) [inline src/pydio/res/i18n/it.po]: Which error? Where from? You might need to keep the `%()s` to specify the string cast, I'm not sure though...

**Comment by 7omate** [comment]
I added a few comments. Thanks for helping :)

**Comment by DepaMarco** [comment]
@7omate Ok, thanks. I think that it's all right now :)

**Comment by 7omate** [comment]
Looks good to me, thanks.

**Pull request merged**